Walker.java

```diff
@@ -41,6 +41,7 @@ public void setLocation(double startX, double startY){
 
     }
 
+    /* takes a step in a random direction in any angle from 0 to 360 degrees */
```

Comment:
/**
     * Takes a step in a random direction.
     */

the above is javadoc comment style